libs/ui/inputs/src/lib/check-toggle/check-toggle.component.html

```diff
@@ -0,0 +1,18 @@
+<label class="inline-flex relative items-start cursor-pointer" [title]="title">
+  <span class="shrink-0">
+    <input
+      type="checkbox"
+      class="sr-only peer"
+      [ngModel]="value"
+      (ngModelChange)="toggle($event)"
+    />
+    <div
+      class="w-11 h-6 bg-gray-200 peer-focus:outline-none peer-focus:ring-2 peer-focus:ring-{{
+        color
+      }}-lighter rounded-full peer peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all peer-checked:bg-{{
```

Comment:
aren't some classes purged from TW ? eg `peer-checked:bg-{{color}}`

also i don't get why you need the `peer` class in the div as well.

Reply:
Good catches, thanks ! Wondering why this was working yesterday. Also, the `peer` can be removed.